packages/utils/src/lib/reports/utils.ts

```diff
@@ -304,3 +305,29 @@ export function compareIssues(a: Issue, b: Issue): number {
 
   return 0;
 }
+
+export function portalCommitLink(
+  config: Pick<UploadConfig, 'project' | 'organization' | 'server'> & {
+    baseUrl?: string;
+  },
+  commit: string,
+): string {
+  const { organization, project, server } = config;
+  // https://portal-api-r6nh2xm7mq-ez.a.run.app/graphql
+  const urlObj = new URL(config.baseUrl || server);
```

Comment:
We don't have a reliable way of creating this link without knowing the portal URL. The API URL is very likely to be different, so trying to derive the path from it will almost certainly produce a broken link.

I'd suggest that we don't interpolate the URL at all and instead print a placeholder `<YOUR_PORTAL_URL>/portal/...`, then at least the user can copy the path and they don't have the friction of a broken link.

If you think it's valuable to provide a portal link, then please create an issue for it - one approach would be to add a property to the config file, another that the server returns the portal link in the response (the 2nd would be my preference).

Reply:
only placed base url there and filled it with `<YOUR_PORTAL_URL>` for now